coverage_assessment.R

```diff
@@ -19,13 +19,38 @@ nut <- get_data()
 
 # Problem 1: Describe the data ----
 
-
-
-
-
-
-
-
+#How many children in the baseline 
+sum(nut$survey_round == "Baseline")
+#Number of children in endline 6552
+sum(nut$survey_round == "Endline")
+nrow (survey_round)
+ncol (survey_round)
```

Comment:
this line will error as well same as above.

Important to remember that `ncol()` is to count the number of columns of a data.frame. There is no `survey_round` object in your environment. `survey_round` is a variable in the `nut` data.frame object.

If you  want to count the number of columns in the `nut` object, use this:

```R
ncol(nut)
```